src/network/NetworkFacade.ts

```diff
@@ -84,7 +84,10 @@ export class NetworkFacade {
     const plainFilePath = filePath;
 
     const encryptedFilePath = fileSystemService.tmpFilePath(`${uuid.v4()}.enc`);
-
+    // Some Android devices doesn't create the file when encrypting, they wait for
+    // the file to exist, since we remove the file and the end anyways
+    // we create an empty one here
+    await fileSystemService.createEmptyFile(encryptedFilePath);
```

Comment:
If we have always the file before encrypting, why is this required?

Reply:
I am not sure if I am understanding this comment properly

Reply:
My bad, this is not needed indeed, we should check if the file exists before the decrypt process starts, fixing